soh/soh/Enhancements/randomizer/hook_handlers.cpp

```diff
@@ -1465,6 +1471,14 @@ void RandomizerOnActorInitHandler(void* actorRef) {
         }
     }
 
+    if (actor->id == ACTOR_BG_TREEMOUTH && LINK_IS_ADULT && IS_RANDO &&
```

Comment:
Should be able to use the `RAND_GET_OPTION` here and get rid of the IS_RANDO (none of this should be registered outside of rando)